src/features/dashboard/components/ApiTokenForm/__tests__/api-token.form.test.tsx

```diff
@@ -214,13 +215,9 @@ describe('Home Page', () => {
       });
 
       const submitButton = screen.getByRole('button', { name: /Create/i });
-
       await act(async () => {
         await userEvent.click(submitButton);
       });
-
-      const modal = await screen.getByText('Your API token is ready to be used.');
-      expect(modal).toBeVisible();
```

Comment:
@agrim-deriv if we remove this expect from this case, then what are we testing here? We need to expect something for each test case.

Reply:
Yes, @shafin-deriv  recognized the change to be made through looking at the test coverages.